Formula/octobuild.rb

```diff
@@ -0,0 +1,23 @@
+class Octobuild < Formula
+  desc "Compiler cache for Unreal Engine"
+  homepage "https://github.com/octobuild/octobuild"
+  url "https://github.com/octobuild/octobuild/archive/refs/tags/0.5.0.tar.gz"
+  sha256 "625fd5bf3cc4ad7f7d93940ab3a12ee0426c3e5388f2a2d8845db03ee2615a01"
+  license "MIT"
+  head "https://github.com/octobuild/octobuild.git", branch: "master"
+
+  depends_on "rust" => :build
+
+  on_linux do
+    depends_on "pkg-config" => :build
+    depends_on "openssl@1.1"
+  end
+
+  def install
+    system "cargo", "install", *std_cargo_args
+  end
+
+  test do
+    system "#{bin}/xgConsole"
```

Comment:
Is there some output we can check here? To see it worked correctly?
```suggestion
    system bin/"xgConsole"
```

Reply:
We can't test the actual work that this tool is supposed to do because that would require a nontrivial amount of set up, but I can test that it outputs some expected informational stuff. Pushed an update.